api/theurlist_localhost_env.json

```diff
@@ -4,7 +4,7 @@
 	"values": [
```

Comment:
Do we need this file anymore?

Reply:
Yes, even though they will be able to use Swagger, we want them to also be bale to use Postman, as the documentation already exists and I updated some of the screenshots as well.